package.json

```diff
@@ -142,6 +142,7 @@
     "husky": "^6.0.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.9.0",
+    "mdx-embed": "^1.0.0",
```

Comment:
enables us to 3rd party content (youtube, vimeo, tweets, codesandbox) in mdx documentation